package.json

```diff
@@ -12,9 +12,9 @@
     "lib": "./lib"
   },
   "dependencies": {
-    "amqp": "0.2.0",
+    "amqp": "0.2.6",
     "async": "0.2.7",
-    "lodash": "2.4.1",
+    "lodash": "4.17.5",
```

Comment:
@damienomurchu Do you know what areas within the code are using lodash? @austincunningham was recently updating lodash for fh-supercore, it would be good to ask what changes were required.